bin/fkCli.py

```diff
@@ -137,70 +137,33 @@ def process_cmd(client: FaaSKeeperClient, cmd: str, args: List[str]):
     return client.session_status, client.session_id
 
 
-@click.command()
```

Comment:
Why all of this code has been removed? The purpose is to extend the existing implementation, not completely replace it.